content/ember-3-24-released.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Ember VER Released
+author:
+  - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+date: 2021-XX-XXT00:00:00.000Z
+tags:
+  - releases
+  - '2021'
+  - version-3-x
+---
+
+Today the Ember project is releasing version VER of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the VER+0.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js VER
+
+Ember.js VER is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+#### Features
+
+Ember.js VER introduced 2 features.
+
+1. Feature description
+2. Feature description
+
+#### Deprecations
+
+Ember.js VER introduced 0 deprecations.
+
+<!-- Block start: If there were no deprecations, remove this block -->
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+<!-- Block end -->
+
+For more details on changes in Ember.js VER, please review the [Ember.js VER.0 release page](https://github.com/emberjs/ember.js/releases/tag/vVER.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
```

Comment:
https://github.com/emberjs/data/blob/v3.24.0/CHANGELOG.md

## Release 3.24.0 (January 4, 2021)
- [#7350](https://github.com/emberjs/data/pull/7350) [Chore]: convert adapters to class syntax
- [#7359](https://github.com/emberjs/data/pull/7359) [CI]: bump failure asset size to 75 bytes
- [#7369](https://github.com/emberjs/data/pull/7369) [CHORE] Bump @ember/ordered-set
- [#7370](https://github.com/emberjs/data/pull/7370) Don't expose the value of a model's attribute in assertions (#7370)
- [#7363](https://github.com/emberjs/data/pull/7363) [BUGFIX identifiers] Address issue with polymorphic findRecord

Reply:
@igorT @snewcomer Hello. I think you helped with the Ember Data 3.24 release?

I noticed that `7363. Address issue with polymorphic findRecord` had been included in the 3.23 release. I was wondering if the changelog might have added a duplicate entry for 3.24. Can you let us know if 7363 should be included in the 3.24 release blog post? Thanks.